javaScript.md

```diff
@@ -3,6 +3,288 @@ title: JavaScript
+      // undefined,
+      {
+        enableHighAccuracy: false,
+        timeout: 60000,
+        maximumAge: 60000
+      }
+    )
+
+```
+
+React.Native persistence requires some setup, best if put in its own file and import at the start of the application
+
+```javascript
+const { AsyncStorage } = require('react-native');
+
+const save = async (key, value) => {
+  await AsyncStorage.setItem(key, value);
+}
+
+const load = async (key) => {
+  return await AsyncStorage.getKey(key);
+}
+
+const remove = async (key) => {
+  return await  AsyncStorage.removeItem(key);
+}
+
+module.exports = {
+  save,
+  load,
+  remove
+}
+
+```
+
+Web setup can done inline like this
+
+```javascript
+matchmore.PlatformConfig.storage = {
+  save: (key, value) => window.localStorage.setItem(key, value),
+  load: (key) => window.localStorage.getItem(key),
+  remove: (key) => window.localStorage.removeItem(key),
+}
+```
+
+Developer is free to mix and match the styles
+
+### Quickstart example
+
+This is an example for web usage of the matchmore sdk
+
+```javascript
+ matchmore.PlatformConfig.storage = {
+              save: (key, value) => window.localStorage.setItem(key, value),
+              load: (key) => window.localStorage.getItem(key),
+              remove: (key) => window.localStorage.removeItem(key),
+            }
+            const localStoragePersistenceManager = new matchmore.LocalStoragePersistenceManager();
+
+            let manager = new matchmore.Manager(
+              apiKey,
+              undefined,
+              localStoragePersistenceManager
+            );
+            manager.createMobileDevice("me", "browser", "").then(device => {
+                manager.startMonitoringMatches();
+                manager.onMatch = (match) => {
+                    writeToScreen(match.publication.properties.name + " matched!");
+                };
+                return device;
+            }).then(device => {
+                //lets wait for the current location
+                let location = new Promise(resolve => {
+                    manager.onLocationUpdate = (loc) => {
+                        writeToScreen("Got Location");
+                        resolve(loc);
+                    };
+                    manager.startUpdatingLocation();
+                    
+                });
+
+                location.then(location => {
+                  const { latitude, longitude, altitude } = location.coords;
+                  const coords = {
+                    latitude,
+                    longitude,
+                    altitude: altitude || 0,
+                  }
+                    let publication = manager.createPinDevice(
+                        "Our test pin",
+                        coords,
+                    ).then(pin => {
+                        let p1 = manager.createPublication(
+                            "my-topic",
+                            100 /* m */,
+                            20 /* s */,
```

Comment:
I like how you added the / m and s . I will keep this idea for the other sdks comment.